scripts/docker/verify.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
-set +x
+set -x
+
+set -eo pipefail
 
 REPO=gcr.io/o1labs-192920
```

Comment:
Are these two variables still relevant? It seems we give the value later in parameter.